requirements.txt

```diff
@@ -7,3 +7,5 @@ netifaces==0.10.9
 pyyaml==5.3.1  # from xivo-lib-python
 sqlalchemy==1.3.22  # from xivo-dao
 sqlalchemy-utils==0.36.8  # from xivo-dao
+psycopg2-binary
```

Comment:
Why is it required ?

And this package is not recommended for production environment. From https://pypi.org/project/psycopg2-binary/ : 

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

Reply:
This requirements.txt is not used for production environment, but for development and testing. debian packaging is used for production environment.

Reply:
It is a dependency of xivo-dao/sqlalchemy for postgresql database.